neuralop/training/torch_setup.py

```diff
@@ -91,6 +91,47 @@ def setup(config):
 
     return device, is_logger
 
+def config_opt_and_scheduler(config, model):
```

Comment:
This looks too opaque to me : can we just have get_optimizer and get_scheduler methods that explicitly list their options?

Reply:
Sure, that sounds simpler